README.md

```diff
@@ -15,7 +15,7 @@ sbt.version=1.2.1
 Add the following to `project/plugins.sbt`:
 
 ```scala
-addSbtPlugin("com.typesafe.sbt" % "sbt-web-build-base" % "1.3.0")
+addSbtPlugin("com.typesafe.sbt" % "sbt-web-build-base" % "1.2.3")
```

Comment:
The version here in the readme was wrong, there is no 1.3.0 yet. I changed it to 1.2.3 which you hopefully will release after merging this pr.